docs/installation/microcontrollers/_index.md

```diff
@@ -160,7 +182,8 @@ If you would like, you can use `mkdir` to initialize a new repository in the dir
 You will be prompted to paste your Viam robot configuration information (`viam.json`) into the terminal.
 
 To obtain this, navigate to [the Viam app](https://app.viam.com).
-Click the **COPY VIAM-SERVER CONFIG** button on the right side of the **SETUP** tab of your robot to copy the text for the config file.
+Click the **COPY VIAM-SERVER CONFIG** button on the right side of the **SETUP** tab of your robot.

```

Comment:
We do say "Navigate to [the Viam app](https://app.viam.com) and create a new robot in your desired location." above. 

Because a new user has an org and a location created by default, that is enough, I think.